daemon/emer-daemon.h

```diff
@@ -73,7 +73,8 @@ struct _EmerDaemonClass
 GType                    emer_daemon_get_type                 (void) G_GNUC_CONST;
 
 EmerDaemon *             emer_daemon_new                      (const gchar             *persistent_cache_directory,
-                                                               EmerPermissionsProvider *permissions_provider);
+                                                               EmerPermissionsProvider *permissions_provider,
+                                                               EmerMachineIdProvider *machine_id_provider);
```

Comment:
Maybe we should use `EmerTrackingIdProvider` and `tracker_id_provider` to name these, since that is the newer name? Same applies to other uses of this name across these changes.

Reply:
I reuse the property `tracking-id-path` in EmerMachineIdProvider, so I follow the same naming rule.